src/sdk/account/utils/toSigner.ts

```diff
@@ -5,24 +5,36 @@ import {
   type EIP1193Provider,
   type EIP1193RequestFn,
   type EIP1474Methods,
+  type Hex,
   type LocalAccount,
   type OneOf,
   type Transport,
   type WalletClient,
   createWalletClient,
-  custom
+  custom,
+  getAddress,
+  hexToBytes
 } from "viem"
 import { toAccount } from "viem/accounts"
 
 import { signTypedData } from "viem/actions"
 import { getAction } from "viem/utils"
 
+// // @Review
+// // This type helps providing other signer types
+// type MinimalSigner = {
```

Comment:
Can we use this and ensure that the type of signers passed into this helper extend this type?

Reply:
Remove comment